gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -425,21 +425,28 @@ message VehiclePosition {
   }
   optional CongestionLevel congestion_level = 6;
 
-  // The degree of passenger occupancy of the vehicle or carriage. This field is still
-  // experimental, and subject to change. It may be formally adopted in the future.
+  // The nominal status of passenger occupancy for the vehicle or carriage.
+  // Individual producers may not publish all OccupancyStatus values. Therefore, consumers
+  // must not assume that the OccupancyStatus values follow a linear scale.
+  // Consumers should represent OccupancyStatus values as the nominal status indicated 
+  // and intended by the producer. Likewise, producers must use OccupancyStatus values that
+  // correspond to actual vehicle occupancy states, and must not interally map OccupancyStatus 
+  // values to interpreted percentages. For describing passenger occupancy levels on a linear 
+  // scale, see `VehiclePosition.occupancy_percentage`.
```

Comment:
Given that this applies to carriage details as well, this should probably be just `occupancy_percentage` instead of `VehiclePosition.occupancy_percentage`.